modules/database/src/std/rec/mbboDirectRecord.c

```diff
@@ -131,16 +139,21 @@ static long init_record(struct dbCommon *pcommon, int pass)
             status = 0;
     }
 
-    if (!prec->udf &&
-        prec->omsl == menuOmslsupervisory) {
-        /* Set initial B0 - B1F from VAL */
-        epicsUInt32 val = prec->val;
+    if (!prec->udf)
+        bitsFromVAL(prec);
+    else {
```

Comment:
I am slightly surprised by the absence of parenthesis for the `if()`.

Reply:
What rule do you think we should follow? Your `bitsFromVAL()` routine defines a `for ()` loop without using braces so you appear to allow some exceptions to a "braces everywhere" rule.

Reply:
Personally, I just keep if and else parts the same.
I do both, but either all without or all with braces.

Reply:
Ditto.

You know I'm not a fan of strict style rules.  I commented because I'd not previously perceived or perpetrated this particular permutation.  (that I know of)

Reply:
Look just below the `CONTINUE:` label in the `process()` routine for a somewhat older example. There's also a similar one where there are braces after the `if ()` but not the following `else if ()` clause immediately above this line.